src/components/Navbar/index.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import './styles.scss';
+import { Link } from 'react-router-dom';
 
 export default function Navbar() {
-  return <nav className="navbar-container">COBDA</nav>;
+  return (
+    <nav className="navbar-container">
+      COBDA
+      <Link to="/sign-up" className="nav-link">
+        Sign Up
+      </Link>
+    </nav>
+  );
 }
```

Comment:
Please add one last line in every file.
```suggestion
}

```